project1.css

```diff
@@ -0,0 +1,188 @@
+header {
+  position: absolute;
+  width: 1340px;
+  height: 1246px;
+  left: 200px;
+  top: 71px;
+}
```

Comment:
Remember to [export the images from Figma](https://help.figma.com/hc/en-us/articles/360040028114-Guide-to-exports-in-Figma) to use them as [background images](https://www.w3schools.com/cssref/pr_background-image.asp) on your page :sunglasses: